site/content/entry/citra-progress-report-2017-july.md

```diff
@@ -0,0 +1,62 @@
++++
+date = "2017-08-01T12:38:00-04:00"
+title = "Citra Progress Report - 2017 July"
+tags = [ "progress-report" ]
+author = "Saphiresurf"
+forum = 0000
++++
+
+With June bringing in many huge improvements to GPU functionality July has come in to sweep up and improve accuracy in a few distinct areas of Cita. Changes and fixes ranging from texture handling all the way to aesthetic changes that help improve the visual appeal and customizability of Citra's QT interface. Now, let's get into what's changed!
+<br />
+
+## [GPU: Fix Edge Cases for TextureCopy](https://github.com/citra-emu/citra/pull/2809) by [wwylele](https://github.com/wwylele)
+
+TextureCopy is a mode of transfer for the GPU activated when the third bit in the GPU's flags register (a register of which is only accessible by the ARM11 processor) is set to 1. When this third bit is switched, the hardware performs a TextureCopy-mode transfer. This particular process ignores all other bits in the gpu's flags register except for bit 2 (which still needs to be set so the GPU knows if it needs to crop the texture or not) and regular dimension registers are ignored. What it's essentially doing is performing a copy of the raw data of the texture, but with a configurable gap in case the texture is going to be copied into a smaller resolution area.
```

Comment:
Do you think saying 

> of which is only accessible by the ARM11 processor

would be less redundant or is it redundant due to something else (possibly that I may not understand about it)?